Src/GUI/Plugins/PlotterPlugin.py

```diff
@@ -128,9 +130,10 @@ def build_panel(self):
             self.datalist = wx.ListCtrl(self.setup, wx.ID_ANY,style = wx.LC_REPORT|wx.LC_SINGLE_SEL)
 
         self.datalist.InsertColumn(0, 'Variable', width=100)
-        self.datalist.InsertColumn(1, 'Axis', width=50)
-        self.datalist.InsertColumn(2, 'Dimension')
-        self.datalist.InsertColumn(3, 'Size')
+        self.datalist.InsertColumn(1, 'Path', width=100)
```

Comment:
I would like to propose a different order for these variables; I think that keeping the order as it has been and just adding the path at the end (last column) is better, given that I don't know  what knowing the path could be useful for. What do you think?